examples/misc/lib/effective_dart/design_bad.dart

```diff
@@ -91,6 +81,18 @@ void miscDeclAnalyzedButNotTested() {
   }
   // #enddocregion inferred-wrong
 
+  {
+    // #docregion redundant-inferred
+    Set<String> things = new Set<String>();
+    // #enddocregion redundant-inferred
+  }
+
+  {
+    // #docregion redundant-explicit
+    var things = new Set();
+    // #enddocregion redundant-explicit
```

Comment:
nit: maybe name this region simply `explicit` (both here and in the `design_good.dart` file), because there is no redundancy if one must be explicit :) (Yes, I know that it is probably because this example comes under the "redundant" guideline.)